app/models/goal.py

```diff
@@ -1,5 +1,25 @@
-from sqlalchemy.orm import Mapped, mapped_column
+from sqlalchemy.orm import Mapped, mapped_column,relationship
```

Comment:
Small spacing issue:
```suggestion
from sqlalchemy.orm import Mapped, mapped_column, relationship
```

The same issue for the same import shows up in the Task model file.

Reply:
updated.thanks